pyasn1_modules/rfc3739.py

```diff
@@ -2,6 +2,8 @@
 # This file is part of pyasn1-modules software.
 #
 # Created by Russ Housley with assistance from asn1ate v.0.6.0.
+# Modified by Russ Housley to add WithComponentsConstraints to
+#   enforce the requirements that are indicated in comments.
```

Comment:
nit: stray indentation

Reply:
This spacing seems right to me, but that is a matter of taste.